content/blog/2020/11/2020-11-11-jira-upgrade-for-the-jenkins-project.adoc

```diff
@@ -0,0 +1,136 @@
+We decided to move from hosting our own Jira server to having the Jira experts at the Linux Foundation host it.
+
+The upgrade timeline looked like this:
+
+* November 2019 - Infrastructure team begins discussions about the November 2020 end of support for Jira 7.13
+* August 2020 - First conversations with Linux Foundation to host Jira for the Jenkins project.  Draft of the upgrade plan assembled and shared with the community
+* September 2020 - Schedule for testing week and final transition week proposed.  Authentication options evaluated and selected
+* October 2020 - Test upgrade performed and tested
+* November 2020 - Final upgrade completed and verified
+
+== Confronting the Complications
+
+Initial discussions between the Jenkins infrastructure team and the Linux Foundation identified complications related to authentication and SSL certificates.
+We planned and negotiated, and tested our assumptions throughout the project.
+
+=== Authentication
+
+Jira servers at the Linux Foundation typically use Linux Foundation accounts for user access.
+Unfortunately, the Jenkins LDAP database includes over 100 000 users.
+It was not feasible to transition 100{nbsp}000 user accounts from the Jenkins LDAP database to the Linux Foundation accounts system and still complete the Jira upgrade before the November 28, 2020 deadline.
+
+The Linux Foundation Project Services team evaluated authentication alternatives and confirmed that they could use the Jenkins LDAP server.
+Using the Jenkins LDAP server spared us from two transitions, LDAP and Jira, and kept the project timeline feasible.
+
+=== SSL Certificates
+
+Jira servers at the Linux Foundation use Let's Encrypt to generate SSL certificates for HTTPS.
+The Linux Foundation uses the DNS method to obtain SSL certificates.
+Unfortunately, the Jenkins project uses the HTTP method to obtain SSL certificates.
+
+Thankfully, Olivier Vernin of the Jenkins project and Anton Baranov of the Linux Foundation found a solution.
+They were able to grant sufficient permissions in the Jenkins DNS records so that the Linux Foundation could issue SSL certificates for issues.jenkins.io and issues.jenkins-ci.org.
+
+=== Building the Prototype
+
+Anton Baranov created a prototype Jira server, restored an older Jenkins Jira backup, and upgraded it to Jira 8.13.
+That first restore detected that we had not provided the Jira attachments or the Jira avatars.
+That attachments and avatars added multiple gigabytes to the initial backup data and were vital to complete the update.
+
+== Testing the Upgrade
+
+A group of volunteers including Jenkins users, security team members, and infrastructure team members tested the upgrade during the week of October 26, 2020.
+The tests confirmed that authentication worked as expected and that the Jira prototype was functioning as expected.
+
+We thank the test team, including:
+
+* Daniel Beck
+* Tim Jacomb
+* Olivier Vernin
+* Mark Waite
```

Comment:
@olblak , @timja , @daniel-beck who did I miss that assisted with upgrade testing?